datastream-kafka-connector/src/test/java/com/linkedin/datastream/connectors/kafka/mirrormaker/TestKafkaMirrorMakerConnector.java

```diff
@@ -221,9 +220,7 @@ public void testValidateDatastreamUpdatePausedPartitions() throws Exception {
     coordinator.initializeDatastream(datastream);
 
     // create topic
-    if (!AdminUtils.topicExists(_zkUtils, topic)) {
-      AdminUtils.createTopic(_zkUtils, topic, 2, 1, new Properties(), null);
-    }
+    createTopic(_adminClient, topic, 2);
```

Comment:
how could we configure the replication and other properties with the newer createTopic method?